CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.12.1 (2024-02-08)
+
+### New Features
```

Comment:
We don't expose new APIs / features to users so "Improvements" might be better than "New Features".